### PR TITLE
Updated text to better describe project and also make wiki link more visible

### PR DESCRIPTION
**Pull request by bsclifton**
Updated text to better describe project and also make wiki link more visible

Auditors: @bbondy, @bridiver, @darkdh, @posix4e

### CODE REVIEW

**Comment by posix4e** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# browser-laptop-bootstrap
+Everything you need to compile [Muon](https://github.com/brave/muon) and run [Brave](https://github.com/brave/browser-laptop)
+- fetches all 3rd party source code (Chromium)
+- checks out [Muon](https://github.com/brave/muon) as a submodule
```
Comment (bridiver) [inline README.md]: none of these are checked out as submodules

File: README.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+- checks out [our fork of node](https://github.com/brave/node/tree/chromium54) as a submodule under Muon
```
Comment (bridiver) [inline README.md]: node is handled in muon DEPS along with several other repos that it requires

**Comment by bsclifton** [comment]
@bridiver thanks for the feedback :smile: I took some time to go through the scripts and really see how it works. Text updated!

**Pull request merged**